<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlurEffectConfig</class>
 <widget class="QWidget" name="BlurEffectConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>444</width>
    <height>250</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>444</width>
    <height>450</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelConstantBlurDescription">
     <property name="text">
      <string>General:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelConstantNoiseDescription">
     <property name="text">
      <string>Blur region mask:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="kcfg_TextureLocation">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browse_pushButton">
       <property name="toolTip">
        <string>Browse...</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="document-open"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_pushButton">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-delete"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>Enable this effect for the following Firefox-based browsers (comma separated):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="kcfg_IncludedWindows">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
